app/components/DR/ButtonUpDrawer.js

```diff
@@ -28,7 +28,7 @@ export default class BottomUpPanel extends Component {
       start: height - this.props.startHeight,
       end: this.props.topEnd,
       min: this.props.topEnd,
-      animates: [() => this.animatedHeight],
+      animates: [() => this._animatedHeight],
```

Comment:
why the _ ??

eslint have the entire file in red for this underscore

![image](https://user-images.githubusercontent.com/43874147/82389804-0b1d2a80-9a0b-11ea-80a2-37e6390d4660.png)

Reply:
Didn't notice, gonna fix it